README.md

```diff
@@ -19,6 +19,7 @@ The _"non-endorsed"_ status means that the plugin is not endorsed by the origina
 | [**camera_tizen**](packages/camera) | [camera](https://github.com/flutter/plugins/tree/master/packages/camera) (1st-party) | [![pub package](https://img.shields.io/pub/v/camera_tizen.svg)](https://pub.dev/packages/camera_tizen) | No |
 | [**connectivity_plus_tizen**](packages/connectivity_plus) | [connectivity_plus](https://github.com/fluttercommunity/plus_plugins/tree/main/packages/connectivity_plus) (1st-party) | [![pub package](https://img.shields.io/pub/v/connectivity_plus_tizen.svg)](https://pub.dev/packages/connectivity_plus_tizen) | No |
 | [**device_info_plus_tizen**](packages/device_info_plus) | [device_info_plus](https://github.com/fluttercommunity/plus_plugins/tree/main/packages/device_info_plus) (1st-party) | [![pub package](https://img.shields.io/pub/v/device_info_plus_tizen.svg)](https://pub.dev/packages/device_info_plus_tizen) | No |
+| [**flutter_blue_tizen**](packages/flutter_blue) | [flutter_blue_tizen](https://github.com/fluttercommunity/plus_plugins/tree/main/packages/flutter_blue) (3rd-party) | [![pub package](https://img.shields.io/pub/v/flutter_blue_tizen.svg)](https://pub.dev/packages/flutter_blue_tizen) | No |
```

Comment:
The second entry in the table refers to the app-facing package, in the case of `fluttter_blue_tizen`, it's `flutter_blue`, it should be: \[flutter_blue](https://github.com/pauldemarco/flutter_blue) (Also note the url difference).